unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneBoundariesController/SceneBoundsChecker.cs

```diff
@@ -171,7 +185,7 @@ public void AddEntityToBeChecked(IDCLEntity entity, bool isPersistent = false, b
 
                 // No need to add the entity to be checked later if we already found it outside scene outer boundaries.
                 // When the correct events are triggered again, the entity will be checked again.
-                if (!isPersistent && !entity.isInsideSceneOuterBoundaries)
+                if (!isInsistent && !isPersistent && !entity.isInsideSceneOuterBoundaries)
```

Comment:
There's a deeper issue here that I didn't manage to fix yet, this flag just disables this optimization in order to avoid invisible objects, the bug that causes those objects to keep being invisible is still out there

Reply:
yep, once the scene is relocated inside, it should be visible again... Weird... This might affect performance, we have to watchout.

Reply:
yep, once the scene is relocated inside, it should be visible again... Weird... This might affect performance, we have to watchout.